connect/api/src/main/java/org/apache/kafka/connect/connector/Connector.java

```diff
@@ -130,6 +131,9 @@ public void reconfigure(Map<String, String> props) {
      */
     public Config validate(Map<String, String> connectorConfigs) {
         ConfigDef configDef = config();
+        if (null == configDef) {
+            throw new ConnectException("Connector.config() must return a ConfigDef.");
```

Comment:
nit: Maybe we may as well include the connector's classname in the exception message like the other cases?

Reply:
+1